packages/date-adapters/src/date-fns-adapter/index.ts

```diff
@@ -352,6 +339,24 @@ export class AdapterDateFns implements SaltDateAdapter<Date, Locale> {
     });
   }
 
+  /**
+   * Get the timezone from the Date object
+   * @param date - A Date object
+   * @returns "default" as Timezones are not supported by the Date object
+   */
+  public getTimezone = (): string => {
+    return "default";
+  };
+
+  /**
+   * Set the timezone for the Day.js object
+   * @param date - A Day.js object
+   * @returns  'UTC' | 'system' or the IANA time zone
+   */
+  public setTimezone = (date: Date): Date => {
```

Comment:
fyi, `date-fns` [timezone support](https://date-fns.org/v4.1.0/docs/Time-Zones) is supported by additional package ([@date-fns/tz](https://github.com/date-fns/tz) and [@date-fns/utc](https://github.com/date-fns/utc))

Reply:
Requires a Typescript update, v5, so not adding in this iteration.
Some folks reported that DatePicker works with v4, however, the adapters now include a setTimezone, which would not work, so you would need to augment the adapter and provide your own setTimezone.

For now, not supporting v4 officially, will look for folks who are using v4 to feedback and then we can update the docs (until such time as we upgrade TS)